src/libraries/TickMath.sol

```diff
@@ -46,7 +46,15 @@ library TickMath {
     /// at the given tick
     function getSqrtPriceAtTick(int24 tick) internal pure returns (uint160 sqrtPriceX96) {
         unchecked {
-            uint256 absTick = tick < 0 ? uint256(-int256(tick)) : uint256(int256(tick));
```

Comment:
could you add 1 test to show that the behaviour for type(int24).min is unchanged - just bc thats the 1 number that performs `-1` weirdly. Something like

```

    function test_getSqrtPriceAtTick_throwsForMinIn24() public {
        vm.expectRevert(TickMath.InvalidTick.selector);
        tickMath.getSqrtPriceAtTick(type(int24).min);
    }
```